.gitignore

```diff
@@ -53,4 +53,7 @@ bin/tmp/*
 scratchpad
 gen.gen
 
-**/*-wal/
\ No newline at end of file
+**/*-wal/
+.devcontainer
+/eudico-stats
```

Comment:
This is because we are collecting the metrics in the current path by default?

Reply:
`/eudico-stats` is the built executable, haha. I just excluded it from being pushed to git.

Reply:
Ah, understood. LGTM.